driver/connect.c

```diff
@@ -3196,6 +3196,8 @@ SQLRETURN EsSQLSetConnectAttrW(
 			RET_HDIAGS(dbc, SQL_STATE_HY092);
 			break;
 
+		case SQL_ATTR_QUERY_TIMEOUT: /* stmt attr -- 2.x */
+			WARNH(dbc, "applying a statement as connection attribute (2.x?)");
```

Comment:
Does this need a `break;`?

Reply:
It does not, all three attributes are mapped to same behavior. I could add a comment if necessary.

Reply:
I think it would be worth adding a comment (or expanding the existing one).  The fact that the existing comment `/* coalesce login and connection timeouts for a REST req. */` doesn't mention the query timeout reinforces the thought that a `break` is missing.